crates/payload/basic/src/lib.rs

```diff
@@ -861,24 +864,23 @@ pub fn pre_block_beacon_root_contract_call<DB: Database + DatabaseCommit, Attrib
     db: &mut DB,
     chain_spec: &ChainSpec,
     block_number: u64,
-    initialized_cfg: &CfgEnv,
+    initialized_cfg: &CfgEnvWithHandlerCfg,
     initialized_block_env: &BlockEnv,
     attributes: &Attributes,
 ) -> Result<(), PayloadBuilderError>
 where
     DB::Error: std::fmt::Display,
```

Comment:
preferable if the trait bound is `std::error::Error`, no?

Reply:
I think this is okay, we have our own types. with standard Error we would just lose information